### ISSUE TITLE
detail page has no h1

### ISSUE THREAD

**Issue by fuzzylogic2000** [liqd/digitalstrategie#435]
**URL:** 
**user:** 
**expected behaviour:** Every page with headlies should also have an h1. So the detail page should have required German page title to ensure that.
**behaviour:** We do not set a page title, but in the paragprah (richtext) blocks, h2, h3, and h4 are settable.
**important screensize:**
**device & browser:** 
**Comment/Question:** @maxliqd @hmkay Should we add a page title? And when that is the decision should we also check that all the other pages have a required one?

Screenshot?

**Comment by maxliqd** [comment]
Yes, I don't see any other way to meet a11y standards as we don't have any h1 elsewhere, right? Thanks!

**Comment by fuzzylogic2000** [comment]
Yeah, we do not have that on this page. I will also check the other pages.

**Comment by maxliqd** [comment]
I agree, let's add the page title in h1 everywhere.
@hmkay I don't think h4 is disturbing me. What do you think?

**Comment by fuzzylogic2000** [comment]
I just thought removing the h4 makes it harder to get it wrong. :) But also fine to keep it.
